### ISSUE TITLE
Minor improvement - Sidebar Button overflows into other elements

### ISSUE THREAD

**Issue by yannikp** [grokability/snipe-it#11754]
### Debug mode

- [x] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

The border of the sidebar-button overflows into other elements and thus shifts them slightly.

I have this custom CSS in place to fix it.
.sidebar-toggle.btn {
	border-radius: 3px;
	box-shadow: none;
	border-top: 0px solid transparent;
	border-bottom: 0px solid transparent;
	padding-left: 15px;
	padding-right: 15px;
	padding-top: 12px;
	padding-bottom: 12px;
	margin-left: -47px;
	margin-top: 2px;
}

### Reproduction steps

Open the Snipe-IT Demo on a Desktop with a resolution of at least 1100px in width and click on "Create New" "Admin".

### Expected behavior

A pristine interface :D

### Screenshots

![grafik](https://user-images.githubusercontent.com/22473767/187190156-926c3cae-abcc-4ff7-aa8b-ba05fae9115c.png)

### Snipe-IT Version

Snipe-IT Demo

### Operating System

Snipe-IT Demo

### Web Server

Snipe-IT Demo

### PHP Version

Snipe-IT Demo

### Operating System

Windows, macOS

### Browser

Chrome, Firefox

### Version

Chrome v104, Firefox v105

### Device

Desktop

**Comment by ccj9874** [comment]
Can confirm that I had this issue on my instance and that the CSS you shared fixed it. I have no clue how you noticed this haha

**Comment by snipe** [comment]
We'd be happy to take a PR for this :)

**Comment by yannikp** [comment]
Understandable ^^
Is it this file where I would need to add the change? _/resources/assets/less/overrides.less_

**Comment by snipe** [comment]
Yep - you don't need to submit compiled assets though, we do that on our end :)
